packages/rest/package.json

```diff
@@ -26,8 +26,6 @@
     "@loopback/context": "^4.0.0-alpha.20",
     "@loopback/core": "^4.0.0-alpha.22",
     "@loopback/openapi-spec": "^4.0.0-alpha.15",
-    "@types/http-errors": "^1.5.34",
```

Comment:
This needs to be in dependencies because we re-export the http-errors objects for use in @loopback/rest.
Removing this will cause compilation failures downstream!